app/src/main/java/com/team7/cmput301/android/theirisproject/controller/EditRecordController.java

```diff
@@ -59,8 +79,21 @@ public boolean submitRecord(String title, String desc){
         record.setDesc(desc);
 
         if (IrisProjectApplication.isConnectedToInternet()) {
+
             new EditRecordTask().execute(record);
+
+            // Add only the new Record Photos.
+            // newRecordPhotos holds ALL photos that were added in activity
+            // (including ones that were added then deleted), while the model
+            // properly removes deleted photos.
```

Comment:
are we deleting the photos in the database? you might have to add a deleteQueue in EditRecordController to send delete requests when you submit the edit. (Maybe change deleteRecord to return the _id once we remove it from the list)